From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 23 Jan 2017 04:28:47 +0000
Subject: memcg: Avoid ABI change in 4.9.5
Forwarded: not-needed

struct mem_cgroup_per_node was changed by commit b4536f0c829c
"mm, memcg: fix the active list aging for lowmem requests when
memcg is enabled", backported into 4.9.5.  This structure
isn't used by modules, so hide the change.

---
--- a/include/linux/memcontrol.h
+++ b/include/linux/memcontrol.h
@@ -120,7 +120,11 @@ struct mem_cgroup_reclaim_iter {
  */
 struct mem_cgroup_per_node {
 	struct lruvec		lruvec;
+#ifdef __GENKSYMS__
+	unsigned long		lru_size[NR_LRU_LISTS];
+#else
 	unsigned long		lru_zone_size[MAX_NR_ZONES][NR_LRU_LISTS];
+#endif
 
 	struct mem_cgroup_reclaim_iter	iter[DEF_PRIORITY + 1];
 
